.github/workflows/lint.yml

```diff
@@ -13,8 +13,8 @@ jobs:
       with:
         node-version: 18
     - name: Install deps
-      run: yarn
+      run: pnpm install
     - name: Run textlint
-      run: yarn lint
+      run: pnpm run lint
```

Comment:
I do not short syntax like `pnpm lint` according to https://github.com/vitejs/vite/pull/5060#discussion_r715237130.